### DISCUSSION TITLE
Need Help finding thread on old forum

### DISCUSSION THREAD

**Discussion by utwsoft** [ImageMagick/ImageMagick#2619] [Help] [answered]
I didn't realize that ImageMagick has locked its servers and moved to github discussions. Can anyone help me find the thread that this link is supposed to go to?

https://www.imagemagick.org/discourse-server/viewtopic.php?t=31926

Navigating there doesn't lead to the thread anymore.

**Comment by snibgo** [comment]
How about: https://legacy.imagemagick.org/discourse-server/viewtopicd4e3.html?f=1&t=31926

I found this by navigating to any thread in the old forum, then changing the address to have the number you supplied, 31926.

**Reply by utwsoft** [reply]
Yeah, I was trying this with no luck. Thank you, I'll probably keep doing that. The summary of the thread is this from my google search:

Finding first and last line and column with non-transparent ...www.imagemagick.org › Magick › Board index › Users
May 14, 2017 - Finding first and last line and column with non-transparent pixels? Questions and postings pertaining to the usage of ImageMagick regardless ...
You've visited this page many times. Last visit: 9/24/20

I guess I can hunt by date now too. A bummer that all that wealth of information is not really accessible without a working search function.

**Reply by utwsoft** [reply]
Or not. Seems like anything between pages 12 and like, 298 are corrupted or something. Oh well. I guess I'll have to rediscover my solution.

**Comment by Dave-Allured** [comment, marked as answer]
Currently you can access old forum discussions with a simple URL hack.  Change "www" to "archive", and you're there.  For example, old and new:

https://www.imagemagick.org/discourse-server/viewtopic.php?t=31926
https://archive.imagemagick.org/discourse-server/viewtopic.php?t=31926

**Reply by utwsoft** [reply]
Yaas! Thank you so much! A few days ago I tried this, but the url was different... There was a few hex digits inbetween viewtopic and php. i.e. viewtopice3bc.php..... or something like that, and I couldn't do this trick. I suppose it was in the middle of migration or crawling, or whatever. Anyway, thank you again, and I'm glad the old archives are being kept intact!
